tests/legacy_comparison/archives.txt

```diff
@@ -0,0 +1,20 @@
+astro-ph
```

Comment:
This will need to be updated independently of the taxonomy whenever there's a new archive.